pkg/virtctl/create/vm/vm_test.go

```diff
@@ -1300,45 +1384,50 @@ chpasswd: { expire: False }`
 	})
 
 	Describe("Manifest is not created successfully", func() {
-		DescribeTable("Invalid arguments to RunStrategyFlag", func(runStrategy string) {
-			out, err := runCmd(setFlag(RunStrategyFlag, runStrategy))
-			Expect(err).To(MatchError(fmt.Sprintf("failed to parse \"--run-strategy\" flag: invalid RunStrategy \"%s\", supported values are: Always, Manual, Halted, Once, RerunOnFailure", runStrategy)))
+		const (
+			invalidInferenceVolumeError   = "inference of instancetype or preference works only with datasources, datavolumes or pvcs"
```

Comment:
wouldn't be better to define these errors in single package, instead of defining it in 2 packages?

Reply:
This was done intentionally to decouple testing from production code. If something changes in production the change is not applied silently to the test as well.